data/remnant ships.txt

```diff
@@ -348,6 +347,87 @@ ship "Gull" "Gull (Bunks)"
 
 
 
+ship "Ibis"
+	sprite "ship/ibis"
+	thumbnail "thumbnail/ibis"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Remnant Capital"
+		"cost" 30000000
+		"shields" 22000
+		"hull" 11000
+		"required crew" 20
+		"bunks" 48
+		"mass" 300
+		"drag" 7.1
+		"heat dissipation" 0.7
+		"fuel capacity" 700
+		"ramscoop" 2
+		"outfit space" 568
+		"cargo space" 103
```

Comment:
Outfit space and cargo space positions are flipped.

Reply:
Fixed.